x-pack/functionbeat/docs/getting-started.asciidoc

```diff
@@ -0,0 +1,224 @@
+
+// REVIEWERS: will the download path include x-path? In BC2, it's 
+// https://staging.elastic.co/6.5.0-92a7b7db/downloads/beats/x-pack/functionbeat/functionbeat-6.5.0-windows-x86_64.zip
+
+[[linux]]
+*linux:*
+
+ifeval::["{release-state}"=="unreleased"]
+
+Version {version} of {beatname_uc} has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+["source","sh",subs="attributes"]
+------------------------------------------------
+curl -L -O https://artifacts.elastic.co/downloads/beats/{beatname_lc}/{beatname_lc}-{version}-linux-x86_64.tar.gz
+tar xzvf {beatname_lc}-{version}-linux-x86_64.tar.gz
+------------------------------------------------
+
+endif::[]
+
+[[mac]]
+*mac:*
+
+ifeval::["{release-state}"=="unreleased"]
+
+Version {version} of {beatname_uc} has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+["source","sh",subs="attributes"]
+------------------------------------------------
+curl -L -O https://artifacts.elastic.co/downloads/beats/{beatname_lc}/{beatname_lc}-{version}-darwin-x86_64.tar.gz
+tar xzvf {beatname_lc}-{version}-darwin-x86_64.tar.gz
+------------------------------------------------
+
+endif::[]
+
+[[win]]
+*win:*
+
+ifeval::["{release-state}"=="unreleased"]
+
+Version {version} of {beatname_uc} has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+. Download the {beatname_uc} Windows zip file from the
+https://www.elastic.co/downloads/beats/{beatname_lc}[downloads page].
+
+. Extract the contents of the zip file. 
+
+endif::[]
+
+[id="{beatname_lc}-configuration"]
+=== Step 2: Configure {beatname_uc}
+
+Before deploying {beatname_uc} to your serverless environment, you need to
+specify details about the functions that you want to deploy, including the
+function name, type, and triggers that will cause the function to execute.
+You also need to specify connection details for your {es} cluster. 
+
+You specify settings in the  +{beatname_lc}.yml+ configuration file. This file
+is located in the archive that you extracted earlier.
+
+TIP: See the
+{libbeat}/config-file-format.html[Config File Format] section of the
+_Beats Platform Reference_ for more about the structure of the config file.
+
+The following example configures a function called `cloudwatch` that collects
+events from CloudWatch Logs and forwards the events to {es}.
+
+["source","sh",subs="attributes"]
+-------------------------------------------------------------------------------------
+{beatname_lc}.provider.aws.deploy_bucket: "functionbeat-deploy"
+{beatname_lc}.provider.aws.functions:
+  - name: cloudwatch
+    enabled: true
+    type: cloudwatch_logs
+    description: "lambda function for cloudwatch logs"
+    triggers:
+      - log_group_name: /aws/lambda/my-lambda-function
+output.elasticsearch:
+  cloud.id: "MyESDeployment:SomeLongString=="
+  cloud.auth: "elastic:SomeLongString"
+processors:
+  - add_host_metadata: ~
+  - add_cloud_metadata: ~
```

Comment:
I would remove the above processors, in the serverless context they are not really useful because you do not have any control on where the lambda will be executed.